index.html

```diff
@@ -0,0 +1,201 @@
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <section class="hero">
+        <div class="hero-text main-hero-txt">
+            <div class="hero-headings ">
+               <div class="first-part first-part-hero">
+                <p>"Hello! Sharing  world"</p>
+                <h1>CREATIVE COMMONS GLOBAL SUBMIT 2015</h1>
+               </div>
+                <p class="second-part second-part-hero ">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+            <div class="date">
+                <h3 class="date-head">2015.10.15(THU) ~ 16(FRI)</h3>
+                <p class="date-second">@ National Museum of Karea, Art Center Nabi and more</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="main-program-container">
+           <div class="main-heading">
+            <h3>Main program</h3>
+            <hr>
+           </div>
+           <div class="main-program-content">
+            <div class="contents">
+                <img src="/images/program_icon_01 1.svg">
+                <h4>Lecture</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_02 1.svg">
+                <h4>Forum</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_03 1.svg">
+                <h4>CC Exhibition</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_04 1 (1).svg">
+                <h4>Workshop</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="contents">
+                <img src="/images/program_icon_05 1.svg">
+                <h4>CC Ignite</h4>
+                <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+            </div>
+            <div class="main-btn none">
+                <a>Join cc global submit 2015</a>
+            </div>
+           </div>
+        <div class="remove see-all">
+            <h3>SEE THE WHOLE PROGRAM</h3>
+        </div>
+        </div>
+    </section>
+    <section class="featured">
+        <div class="featured-title">
+            <h3>Featured speakers</h3>
+            <hr>
+        </div>
+        <div class="featured-container">
+        </div>
```

Comment:
- [ ] In higher widths, the "Featured Speakers" section appears like so, kindly fix it.
![image](https://github.com/olanikegloria/capstone-project/assets/62290545/e6cf586b-0d77-4507-97cd-5be11d969939)